### DISCUSSION TITLE
Custom anomaly_maps

### DISCUSSION THREAD

**Discussion by davy-blavette** [open-edge-platform/anomalib#2360] [Q&A]
I don't really understand all these burrs on the heatmap, compratively with a classification I can generate this....is it possible to generate its own heatmap, currently I use a callback 

```
def on_test_batch_end(self, trainer, pl_module, outputs, batch, batch_idx):
        images = batch[‘image’]
        image_paths = batch[‘image_path’]]
        anomaly_maps = outputs[‘anomaly_maps’]
```

but I would like access to the anomaly_maps property, or to be able to generate it myself.
Anomalib
![image](https://github.com/user-attachments/assets/5450b629-3278-4f54-a232-40defc64dc9f)
My heatmap
![image](https://github.com/user-attachments/assets/48a7294b-69d5-440a-82f1-09ab53f2f773)

**Comment by alexriedel1** [comment]
> I don't really understand all these burrs on the heatmap, compratively with a classification I can generate this.

What do you mean by burrs? Where is the actual anomaly in your image that you want to detect? What do you mean by generating this with a classification?
